server/belga/io/feeding_services/twitter_belga.py

```diff
@@ -0,0 +1,124 @@
+from superdesk.io.registry import register_feeding_service
+from superdesk.metadata.item import GUID_FIELD
+
+
+class IngestTwitterBelgaError(SuperdeskIngestError):
+    _codes = {
+        6300: "Invalid iframely api key"
+    }
+
+    @classmethod
+    def TwitterInvalidIframelyKey(cls, exception=None, provider=None):
+        return IngestTwitterBelgaError(6300, exception, provider)
+
+
+class TwitterBelgaFeedingService(TwitterFeedingService):
+    NAME = 'twitter_belga'
+
+    label = 'Twitter Belga'
+
+    ERRORS = [
+        IngestTwitterError.TwitterLoginError().get_error_description(),
+        IngestTwitterError.TwitterNoScreenNamesError().get_error_description(),
+        IngestTwitterBelgaError.TwitterInvalidIframelyKey().get_error_description(),
+    ]
+
+    fields = [
+        {
+            'id': 'consumer_key', 'type': 'text', 'label': 'Twitter Consumer Key',
+            'placeholder': 'Twitter consumer_key', 'required': True,
+            'errors': {6100: 'Twitter authentication failure'}
+        },
+        {
+            'id': 'consumer_secret', 'type': 'password', 'label': 'Twitter Consumer Secret',
+            'placeholder': 'Twitter consumer_secret', 'required': True
+        },
+        {
+            'id': 'access_token_key', 'type': 'text', 'label': 'Twitter Access Token Key',
+            'placeholder': 'Twitter access_token_key', 'required': True
+        },
+        {
+            'id': 'access_token_secret', 'type': 'password', 'label': 'Twitter Access Token Secret',
+            'placeholder': 'Twitter access_token_secret', 'required': True
+        },
+        {
+            'id': 'screen_names', 'type': 'text', 'label': 'Twitter Screen Names',
+            'placeholder': 'Twitter screen_names', 'required': True,
+            'errors': {6200: 'No Screen names specified'}
+        },
+        {
+            'id': 'embed_tweet', 'type': 'boolean', 'default': False, 'label': 'Embed the tweet in the body'
+        },
+        {
+            'id': 'iframely_key', 'type': 'password', 'label': 'iframely API Key',
+            'placeholder': 'iframely api key', 'required': True,
+            'errors': {6300: 'Invalid iframely api key'}
+        },
+    ]
+
+    def _test(self, provider):
+        super()._update(provider, None, test=True)
+        config = provider.get('config', {})
+        key = config.get('iframely_key')
+        self._create_embed('https://iframely.com', key)
+
+    def _update(self, provider, update, test=False):
+        items = super()._update(provider, update, test)[0]
+        config = provider.get('config', {})
+        key = config.get('iframely_key')
+        embed = config.get('embed_tweet')
+        ingest_service = superdesk.get_resource_service('ingest')
+        for item in items:
+            # avoid update_ingest mark item as expired
+            item['versioncreated'] = datetime.now()
+            urls = re.findall(r'http[s]?://(?:[a-zA-Z]|[0-9]|[$-;]|[\[\]?@_~]|'
```

Comment:
No, there isn't. From my experiments, Twitter converts all links to its shorten urls (`t.co`)

Reply:
More explains:
The body html only contain the twitter shorten link (`t.co`), Because Twitter converts all type links (youtube, facebook) to its shorten urls.